app/policies/registry_policy.rb

```diff
@@ -0,0 +1,17 @@
+# RegistryPolicy implements the authorization policy for methods inside the
+# "registry" type.
+class RegistryPolicy
+  attr_reader :user
+
+  def initialize(user, _)
+    raise Pundit::NotAuthorizedError, "must be logged in" unless user
+    @user = user
+  end
+
+  # This method defines the permissions for the following scope string
+  # "registry:catalog:*". Only the "portus" special user is allowed to perform
+  # this call.
+  def all?
+    user.username == "portus"
+  end
```

Comment:
I think we shouldn't limit that to the `portus` user. We should allow all admin users to perform that.

Portus is not offering an API that exposes this information, so 3rd party apps/scripts have to rely distribution's API. Hence we should allow users to run their custom scripts against a distribution instance that uses portus for authentication.